cg/cli/demultiplex/demux.py

```diff
@@ -74,17 +79,15 @@ def demultiplex_flow_cell(
     """
 
     LOG.info(f"Running cg demultiplex flow cell, using {bcl_converter}")
-    flow_cell_directory: Path = Path(context.demultiplex.run_dir, flow_cell_name)
+    flow_cell: Path = Path(context.demultiplex.run_dir, flow_cell_name)

```

Comment:
```suggestion
    flow_cell_dir: Path = Path(context.demultiplex.run_dir, flow_cell_name)

```

Reply:
I think it is already clear by context that it is a dir